eigrpd/eigrp_filter.c

```diff
@@ -74,6 +74,8 @@ void eigrp_distribute_update(struct distribute *dist)
 	/* if no interface address is present, set list to eigrp process struct
 	 */
 	e = eigrp_lookup();
+	if (e == NULL)
```

Comment:
I believe this should be an assert, as that if we get here and we do not ahve a eigrp instance, something has gone so terribly wrong that we are going to crash here shortly anyways.